src/components/input-group/daikin-input-group.ts

```diff
@@ -80,13 +81,17 @@ export class DaikinInputGroup extends LitElement {
   @queryAssignedElements({ selector: "daikin-text-input" })
   _textInputs!: DaikinTextInput[];
 
+  @queryAssignedElements({ selector: "daikin-textarea" })
+  _textareas!: DaikinTextarea[];
```

Comment:
If there is no need to distinguish between the two, then merge them into one.

```suggestion
  @queryAssignedElements({ selector: "daikin-text-input,daikin-textarea" })
  _controls!: ControlElement[];
```

where

```ts
type ControlElement = DaikinTextInput | DaikinTextarea;
```

Reply:
@yodas7 
I completely forgot, but the following line could be used in the logic for displaying the counter (which I moved to `_handleSlotChange`).

I fixed the line that used `getElementByTagName`, leaving the following description. Is this incorrect?

````
@queryAssignedElements({ selector: "daikin-textarea" }
````

Reply:
I think that's correct.
In that case, it is a good idea to have both `_controls` and `_textareas`.